spec/features/common/primary_navigation_spec.rb

```diff
@@ -49,30 +49,32 @@
       expect(incoming_cases_page.primary_navigation.active_link[:href])
         .to eq '/cases/incoming'
       nav_links = incoming_cases_page.primary_navigation.all_links
-      expect(nav_links.count).to eq 6
-      expect(nav_links[0]).to have_text('New cases')
-      expect(nav_links[1]).to have_text('Cases')
-      expect(nav_links[4]).to have_text('Search')
-      expect(nav_links[5]).to have_text('Team')
+      expect(nav_links.count).to eq 7
+      expect(nav_links[0]).to have_text('Create case')
+      expect(nav_links[1]).to have_text('New cases')
+      expect(nav_links[2]).to have_text('Cases')
+      expect(nav_links[5]).to have_text('Search')
+      expect(nav_links[6]).to have_text('Team')
```

Comment:
The numbering looked weird before, and it looks weird now only in a slightly different way :)   - any idea why that was, @digitalali-moj ? Why isn't it sequential? Is there something we're meant to have and don't for accessibility or something?

Reply:
Thanks for adding the missing items. The sequence still matches the if you account for the hidden elements, which can be seen with CSS turned off.